renderer/renderer.js

```diff
@@ -36,8 +36,8 @@ const addFiles = e => {
     title: "Select files to convert",
     filters: [
       { name: "Images", extensions: ["jpg", "png", "jpeg", "webp", "tiff", "bmp"] },
-      { name: "Audio", extensions: ["mp3", "pcm", "wav"] },
-      { name: "Video", extensions: ["mkv", "avi", "mp4", "wmv", "webm"] },
+      { name: "Audio", extensions: ["mp3", "wav", "ogg", "pcm"] },
+      { name: "Video", extensions: ["mkv", "avi", "mp4", "wmv", "webm", "mov"] },
```

Comment:
Instead of having the list of extensions duplicated twice, we can just merge Audio and Video and use the same variable (AUDIO_VIDEO_FORMATS). Maybe the same can be done for the image extensions?

Reply:
Yes, this should be good. We could name it as "Audio/Video" instead

Reply:
Hmm.. Bundling them all together would make the UX simple as the person wouldn't have to bother much when the pop up opens. I think it makes sense, should I bundle all formats into one?

Reply:
> should I bundle all formats into one?
Do you mean images as well?

Reply:
We could just put the "All files" option with the the formats concatenated together.